02.calendar/cal.rb

```diff
@@ -0,0 +1,33 @@
+require 'date'
+require 'optparse'
+
+opt = OptionParser.new
+
+params = {}
+
+opt.on('-y [VAL]') {|v| v }
+opt.on('-m [VAL]') {|v| v }
+
+opt.parse!(ARGV, into: params)
+
+if params[:y].nil?
+  year = Date.today.year
+  month = params[:m].nil? ? Date.today.month : params[:m].to_i
+else
+  year = params[:y].to_i
+  month = params[:m].to_i
+end
+
+month_beginning = Date.new(year, month, 1)
+month_end = Date.new(year, month, -1)
+
+puts "#{month}月 #{year}".center(21) 
+puts " 日 月 火 水 木 金 土"
+
+default_output_width = 3
+printf("%#{month_beginning.wday * default_output_width}s", "")
```

Comment:
printfでがんばろうとすると、記号がたくさん出てきてちょっと読みづらいかな？と思うので、こんなふうに書いた方が見た目にスッキリするかもしれません。

```suggestion
print " " * month_beginning.wday * default_output_width
```

Reply:
こちらも修正しました。printにしたことで見やすくなりました！